framework/src/Volo.Abp.Http.Client/Volo/Abp/Http/Client/ClientProxying/ClientProxyBase.cs

```diff
@@ -256,7 +256,7 @@ await response.Content.ReadAsStringAsync()
                     new RemoteServiceErrorInfo
                     {
                         Message = response.ReasonPhrase,
-                        Code = response.StatusCode.ToString(),
+                        Code = $"{(int)response.StatusCode}",
```

Comment:
Hi,  can you explain why we need this change?

Reply:
response.StatusCode.ToString() returning Http Response enum remark ("Forbidden" for 403) . RemoteServiceErrorInfo.Code need status code integer value.

Localizations work with status code integer values. 
Like that:
```
    "UnhandledException": "Unhandled exception!",
    "401Message": "Unauthorized",
    "403Message": "Forbidden",
    "404Message": "Page not found",
    "500Message": "Internal Server Error",
```
https://github.com/abpframework/abp/blob/ddb36d31d8332450c6299cede04e94a104bb81bc/framework/src/Volo.Abp.ExceptionHandling/Volo/Abp/ExceptionHandling/Localization/en.json#L20

#### Currently, does not show localized exception messages.

Reply:
Hi, thanks. but it is not designed this way.
They are actually for use with Blazor.

<img width="723" alt="image" src="https://github.com/user-attachments/assets/7bb4a57b-8b02-4a1d-9a8a-ed226544d1ac" />

For clients, they are checked by HTTP status code and use these localized texts

https://github.com/abpframework/abp/blob/7c93b7cca65fb803998438b311b800246ac53730/npm/packs/jquery/src/abp.jquery.js#L275-L280

I see that for tiered soultion the error message is not localized, but it has nothing to do with this. can you undo this change? we will check it in the next milestone

Reply:
Ok.